### PR TITLE
Add documentation for some internal function

### PR DESCRIPTION
**Pull request by armano2**

### CODE REVIEW

File: tests/lib/rules/name-property-casing.js [hunk 1]
```diff
@@ -83,8 +93,12 @@ ruleTester.run('name-property-casing', rule, {
           name: 'foo  bar'
         }
       `,
-      options: ['PascalCase'],
```
Comment (mysticatea) [inline tests/lib/rules/name-property-casing.js]: Is this removal intentional?

Comment (armano2) [inline tests/lib/rules/name-property-casing.js]: yes, `PascalCase` is set by default and we was missing test case without options set

Comment (mysticatea) [inline tests/lib/rules/name-property-casing.js]: OK, thank you!
